dayrepo/snippets/static/js/process_add.js

```diff
@@ -0,0 +1,9 @@
+var i = 1 ;
+function addForm() {
```

Comment:
[IMO]
```suggestion
const addForm = () => {
```

必須ではないのですが、今回の関数名 addForm はかなり抽象度が高く命名被りをする可能性があるため、このような記載にしたいです。
基本的な理由は変数と同じです。念の為、修正後動作確認もお願いします。
参考記事は以下
https://jsprimer.net/basic/function-declaration/#function-overwrite